mainmatter/conclusion/futureworks.tex

```diff
@@ -19,67 +19,74 @@ \subsection*{REST API}
 An example here would be the Week Schedule model and endpoint that we have developed.
 Our model uses a class \texttt{WeekSchedule} with an \texttt{enummap} to map \texttt{Weekdays}, where the app currently uses nested sequences instead.
 Either the data retrieved and sent to the REST API must enter a translation process or the app should undergo a refactoring process as to use something that more closely resembles the model used by the REST API.
 The reason for the difference is that we were dissatisfied with the degraded model used by the app, and thus decided to improve the model to retain a high code quality in the REST API.
 As for choosing between one or the other, refactoring may take longer and be the best long term solution, while a translation module may be a faster solution, but worse in the long term and make it harder for future developers to use.
 
-Aside from this, client--side must also consider version control, particular in regards to offline mode and here implement either the solution that we decided would be best in \myref{ssec:policy}, or decide upon another way of handling this issue, either way it must be handled with the use of the REST-API.\todo{Har vi ikke tidligere snakket om at konflikter skal håndteres af serveren? --- MSM}
-\kim{I now understand the problem, how would a solution to this problem look like.}
+As aforementioned some Client--Side implementation may need to consider a secondary storage, like local, in the case they at some point become standalone apps which do not use the login system.
 
 \subsubsection*{Web Administration}
 Once a working REST API has been established, web administration should be implemented concurrently with the client--side implementations.
 In its current state the only way to add users, departments or any other administrative information is by database access, this should be changed such that the department can add new citizens or guardians when needed.
 The reason we believe that a web tool would be the way to solve this, is because the customers has asked for this to be available on their computers, and having it as a web application would made it a simple task to adjust the web application to support tablets should they at a later point want access from such devices.
 \kim{This section is going the right direction, try to elaborate on the problem side a bit more. Also add some technical details to the solution part. Describe how it should you the API, what challenges need to be solved etc. }
 
+In developing such a web administration tool one must not only consider what incoming requests much be supported, but how are the tablets affected by changes through the web administration tool.
+One might also want to consider how the web tool should work, and should it have its own endpoint or should each endpoint support web administration services where it is necessary which leads to figuring out when is it necessary.
```

Comment:
Web admin skal jo bruge de allerede ekstisterende endpoints... so idk what this is :dancers: